modules/nf-core/staramr/search/main.nf

```diff
@@ -0,0 +1,67 @@
+process STARAMR_SEARCH {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/staramr:0.10.0--pyhdfd78af_0':
+        'biocontainers/staramr:0.10.0--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(genome_fasta) // genome as a fasta file
+
+    output:
+    tuple val(meta), path("*_results/results.xlsx")           , emit: results_xlsx
+    tuple val(meta), path("*_results/summary.tsv.gz")         , emit: summary_tsv
+    tuple val(meta), path("*_results/detailed_summary.tsv.gz"), emit: detailed_summary_tsv
+    tuple val(meta), path("*_results/resfinder.tsv.gz")       , emit: resfinder_tsv
+    tuple val(meta), path("*_results/plasmidfinder.tsv.gz")   , emit: plasmidfinder_tsv
+    tuple val(meta), path("*_results/mlst.tsv.gz")            , emit: mlst_tsv
+    tuple val(meta), path("*_results/settings.txt.gz")        , emit: settings_txt
+    tuple val(meta), path("*_results/pointfinder.tsv.gz")     , emit: pointfinder_tsv, optional: true
```

Comment:
Do these files need to be compressed? 

I thought it was was only large files (i.e. sam, fasta, fastq) that needed to be compressed.

Reply:
Technically, not compressed. I wasn't actually sure if all outputs needed to be compressed. If it's only large files I can remove the compression code.

Reply:
ah yes right, I misunderstood earlier when you asked about that. Compression would be needed only for bigger files e.g. fastq

Reply:
Ah, okay. Thanks so much 😄 . Removed gzipping small output files in 33f0f6e68de39a35c9a3d7dde30cc2872f100663

Reply:
Woops. This commit is also needed for removing gzip: 33f0f6e68de39a35c9a3d7dde30cc2872f100663

I made those changes in my editor to `main.nf` and forgot to save the file and commited everything else 😅